Makefile

```diff
@@ -15,7 +15,9 @@ GREEN=`tput setaf 2`
 RESET=`tput sgr0`
 YELLOW=`tput setaf 3`
 
-CODE_QUALITY_VERSION=1.0.1
+# See https://github.com/plone/code-quality
+# Our configuration is in pyproject.toml.
+CODE_QUALITY_VERSION=2.0.0
 LINT=docker run --rm -v "$(PWD)":/github/workspace plone/code-quality:${CODE_QUALITY_VERSION} check
 FORMAT=docker run --rm -v "$(PWD)":/github/workspace plone/code-quality:${CODE_QUALITY_VERSION} format
```

Comment:
Please consider changing this to also use the uid and gid of the current user:
```
CURRENT_USER=$$(whoami)
USER_INFO=$$(id -u ${CURRENT_USER}):$$(getent group ${CURRENT_USER}|cut -d: -f3)
LINT=docker run --rm -v "$(PWD)":/github/workspace plone/code-quality:${CODE_QUALITY_VERSION} check
FORMAT=docker run --user="${USER_INFO}" --rm -v "$(PWD)":/github/workspace plone/code-quality:${CODE_QUALITY_VERSION} format
```

Reply:
Why? Does this fix the permission problems that you mentioned, where Docker would use a different user/group id so you could no longer write a file afterwards?

`getent` does not exist on Mac. Looks like `id -g maurits` works both on Mac and Linux. I will use that and see if the action succeeds. Does that command work for you?

I tried it with a different user, editing the Makefile to have `CURRENT_USER=nika` instead of my own user, and made some changes in a file that should be fixed by `make format`. This worked fine, the file got fixed but was still owned by me afterwards. So either I am lucky, or this is not needed on Mac. Let's see how the checks go.

Reply:
The gh-actions check passes, but this says nothing: `make format` is not called, only `make lint`...